notebooks/getting-started.ipynb

```diff
@@ -15,15 +15,13 @@
    "source": [
     "### 1. Set environment parameters\n",
     "\n",
-    "Be sure to set `env_name` to the name of the Unity environment file you want to launch. Ensure that the environment build is in the `python/` directory."
+    "Be sure to set `env_name` to the name of the Unity environment file you want to launch. Ensure that the environment build is in the ML-Agents SDK directory."
```

Comment:
Shouldn't it be built to the root of the repo?

Reply:
Ah, sorry, I should either rename that to ML-Agents Toolkit or "root directory."